src/components/element-card/element-card.tsx

```diff
@@ -85,10 +86,17 @@ export const ElementCard = ({ element, onClick }: IProps) => {
         showToast(newToast);
     };
 
+
     const handleDeleteClick = (event: React.MouseEvent<HTMLButtonElement>) => {
         event.stopPropagation();
         // TODO: Add confirmation modal
```

Comment:
help me remove this `TODO` comment as we confirmed there wont be a confirmation modal